lib/type/unit/Unit.js

```diff
@@ -977,7 +977,6 @@ function factory (type, config, load, typed, math) {
             }
           }
         }
-        var util = require('util');
```

Comment:
This line is the only reference to `util` in the file - and the `var util` is not used afterward - so it seems safe to remove, and all tests still pass.